merlin/systems/dag/ops/feast.py

```diff
@@ -71,18 +112,48 @@ def from_feature_view(cls, store, path, view, column, output_prefix=None, includ
 
     def __init__(
         self,
-        repo_path,
-        entity_id,
-        entity_view,
-        entity_column,
-        features,
-        mh_features,
-        input_schema,
-        output_schema,
-        include_id=False,
-        output_prefix="",
-        suffix_int=1,
+        repo_path: str,
+        entity_id: str,
+        entity_view: str,
+        entity_column: str,
+        features: List[str],
+        mh_features: List[str],
+        input_schema: Schema,
+        output_schema: Schema,
+        include_id: bool = False,
+        output_prefix: str = "",
+        suffix_int: int = 1,
     ):
+        """
+        Create a new QueryFeast operator to handle link between tritonserver ensemble
```

Comment:
Maybe "...operator to handle the link between an ensemble for Triton Inference Server and a Feast feature store."

The "will" in the second sentence is bugging me.  If neither of the foll two is true, just leave it:

A) "This operator automatically creates your feature store as well as the QueryFeast operator."

B) "You can use the operator to create your feature store as well as the QueryFeast operator."